scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,154 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from typing import Union
+
+from cognite.client.data_classes import TimeSeries, TimeSeriesList
+from cognite.client.exceptions import CogniteNotFoundError
+from cognite.client import CogniteClient
+from cognite.powerops import PowerOpsClient
+
+from cognite.powerops.client._generated.v1.data_classes import BidMatrixInformationWrite
+import random
+import arrow
+
+
+def create_time_series_objects():
```

Comment:
**note (for-future-reference):** some functions are missing complete type hints that include the arguments and return types, so good habit always to write it as some repos will have linting checks that are very strict and require this (it also acts as part of the documentation)